src/main/com/example012/v1/Tickets.java

```diff
@@ -0,0 +1,93 @@
+package main.com.example012.v1;
```

Comment:
Psal jsem přímo, co by měl ticket mít ... 
vytvoř si package model a v něm budeš mít prvně
Company anebo to na co to přejmenuješ
Ticket
a pak uvidíme, co s tím dál. Ten model Ticket je jen objekt, co drží hodnoty ... takže žádná logika. to by měla být nějaká service, co to bude dělat

Reply:
Zkusil jsem to co nejvíce přiblížit MVC a udělal většinu od základu, tak snad to už bude lepší. Pořád mě trápí "magic number" u int[] correctNumbers = new int[lottery.getMaxGuessedNumbers() + 1]; Vím, že bych je neměl používat, ale pole slouží pro počet správných odpovědí které mají 0 - 6 správných (např. 6, ale podle typu hry), maxGuessedNumber je 6, tedy bez +1 nikdy nemůže "úplná výhra" nastat, uniká mi jiný způsob jak to vyřešit, kromě +1 a nebo další proměnné, která by měla adekvátní hodnotu ?